src/autoscaler/pruner/config/config_test.go

```diff
@@ -380,8 +372,8 @@ lock:
 				conf.Lock.ConsulClusterConfig = ""
 			})
 
-			It("should error", func() {
-				Expect(err).To(MatchError(MatchRegexp("Configuration error: Consul Cluster Config is empty")))
+			It("should validate successfully", func() {
+				Expect(err).NotTo(HaveOccurred())
```

Comment:
Since the partial config test will take care of the case "no consul config" so we can remove this test completely.